infra/networks/main.tf

```diff
@@ -93,3 +93,32 @@ resource "aws_vpc_endpoint" "aws_service" {
   subnet_ids          = data.aws_subnets.default.ids
   private_dns_enabled = true
 }
+
+# VPC Configuration for DMS
+# ----------------------------------------
+
+data "aws_ssm_parameter" "dms_peer_owner_id" {
+  name = "/network/dms/peer-owner-id"
+}
+
+data "aws_ssm_parameter" "dms_peer_vpc_id" {
+  name = "/network/dms/peer-vpc-id"
+}
+
+resource "aws_vpc_peering_connection" "dms" {
+  peer_owner_id = data.aws_ssm_parameter.dms_peer_owner_id.value
+  peer_vpc_id   = data.aws_ssm_parameter.dms_peer_vpc_id.value
+  vpc_id        = data.aws_vpc.default.id
+  peer_region   = "us-east-2"
+
+  tags = {
+    Name = "DMS VPC Peering"
+  }
+}
+
+resource "aws_route" "dms" {
```

Comment:
Note for posterity: After discussions with MicroHealth, we acknowledged the potential need to adopt a custom route table with explicit subnet associations to enhance network segmentation and control. Currently, we're utilizing the default route table, which is implicitly associated with all subnets within the VPC.